src/gobstuf/stuf/message.py

```diff
@@ -65,6 +65,39 @@ def set_elm_value(self, elements_str: str, value: str, tree=None):
         elm = self.find_elm(elements_str, tree)
         elm.text = value
 
+    def create_elm(self, elements_str: str, tree=None):
+        """Creates an element when it doesn't exist yet and returns newly created element. If the element already
+        exists, the existing element is returned.
+
+        :param elements_str:
+        :param tree:
+        :return:
+        """
+        elm = self.find_elm(elements_str, tree)
+
+        if elm is not None:
+            # Already exists
```

Comment:
Isn't if elm just enough here? Or is it just to be in line with the is None test?

Reply:
No. If elm is empty we still want to return it. (Otherwise we would be creating a sibling of the existing empty element)